storage/memory/static_test.go

```diff
@@ -173,10 +174,14 @@ func TestStaticPasswords(t *testing.T) {
 			},
 		},
 		{
-			name: "create password",
+			name: "create passwords",
 			action: func() error {
-				return s.CreatePassword(p3)
+				if err := s.CreatePassword(p3); err != nil {
+					return err
+				}
+				return s.CreatePassword(p4)
```

Comment:
do you want me to check if "spam@example.com" gets created with the all lower case?